src/plugins/telemetry/server/config/telemetry_labels.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+
+/**
+ * Labels to enrich the context of the telemetry generated.
+ * When adding new keys, bear in mind that this info is exposed
+ * to the browser **even to unauthenticated pages**.
+ */
+export const labelsSchema = schema.object(
+  {
+    branch: schema.maybe(schema.string()),
+    ciBuildJobId: schema.maybe(schema.string()),
+    ciBuildId: schema.maybe(schema.string()),
+    ciBuildNumber: schema.maybe(schema.number()),
+    ftrConfig: schema.maybe(schema.string()),
+    git_rev: schema.maybe(schema.string()),
+    isPr: schema.maybe(schema.boolean()),
+    prId: schema.maybe(schema.string()),
+    journeyName: schema.maybe(schema.string()),
+    testBuildId: schema.maybe(schema.string()),
+    testJobId: schema.maybe(schema.string()),
```

Comment:
What is the difference between `testBuildId` and `ciBuildId`? The same question applies to `*JobId` fields 😇

Reply:
testJobId and testBuildId is for development purposes, global labels such as ciBuildId and ciJobId does not have value when we are running the tests locally however testJobId and testBuildId has always a value which is something like this `testBuildId=ciBuildId || 'local'+uuid()` 😊